Monika After Story/game/zz_backgrounds.rpy

```diff
@@ -337,21 +340,40 @@ init 800 python:
                 The background we're changing to
 
             by_user:
-                If the user switched the background themselves
+                True if the user switched the background themselves
+
+            set_persistent:
+                True if we want this to be persistent
         """
         if by_user is not None:
             mas_background.force_background = bool(by_user)
 
         mas_current_background.exit(new_background)
         mas_setBackground(new_background)
 
+    def mas_startupBackground():
+        """
+        Sets up the spaceroom to start up in the background you left in if it is unlocked and still exists
+        """
+        if (
+            persistent._mas_current_background in store.mas_background.BACKGROUND_MAP
+            and store.mas_background.BACKGROUND_MAP[persistent._mas_current_background].unlocked
+        ):
+            background_to_set = store.mas_background.BACKGROUND_MAP[persistent._mas_current_background]
+            mas_changeBackground(background_to_set)
+
+            if background_to_set.disable_progressive:
+                store.skip_setting_weather = True
+
     #Just set us to the normal room here
     mas_current_background = None
     mas_setBackground(mas_background_def)
 
     #Make sure the bg selector is only available with at least 2 bgs unlocked
     if mas_background.getUnlockedBGCount() < 2:
         mas_lockEVL("monika_change_background","EVE")
+    else:
+        mas_unlockEVL("monika_change_background", "EVE")
```

Comment:
bg selector topic locking/unlocking should happen in `ch30_reset` instead, in case a late update script/submodl/delayed action/etc makes a change beyond init level 800.